Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -197,6 +197,55 @@ final class LayerTests: XCTestCase {
         XCTAssertEqual(output, expected)
     }
 
+    func testConv3DGradient() {
+        let filter =  Tensor(shape: [1, 4, 4, 1, 1], scalars: (0..<16).map(Float.init))
```

Comment:
Please make sure to use proper spacing or run swift-format before pushing.

```suggestion
        let filter = Tensor(shape: [1, 4, 4, 1, 1], scalars: (0..<16).map(Float.init))
```